backend/python/app/graphql/types/story_type.py

```diff
@@ -33,3 +33,18 @@ class StoryRequestDTO(graphene.InputObjectType):
     youtube_link = graphene.String(required=True)
     level = graphene.Int(required=True)
     translated_languages = graphene.List(LanguageEnum)
+
+
+class NewStoryTranslationResponseDTO(graphene.ObjectType):
```

Comment:
nit: can we name these `CreateStoryTranslationResponseDTO` etc. Keeps in convention with other creation DTOS